src/components/location/Location.js

```diff
@@ -0,0 +1,84 @@
+import './location.scss';
+import $ from 'jquery';
+
+class GoogleMap {
+  constructor(container, position) {
+    this.container = container;
+	  this.position = position;
+    this._googleMapKey = 'AIzaSyBApIxlKjCG4nPYE2GnZa6UltnP8JqMdgA';
+    this.zoom = 15;
+    this.checkPosition();
+  }
+
+  checkPosition() {
+    if(this.position instanceof Function) {
+      this.position().then((pos) => {
+        this.position = pos;
+      })
+    }
+  }
+	
+	changePosition(position) {
+		this.map.panTo(position);
+		this.marker.setPosition(position);
+	}
+	
+	initMap = () => {
+    $.getScript('https://maps.googleapis.com/maps/api/js?key=' + this._googleMapKey + '', this._setMap);
+	}
+	
+	_setMap = () => {
+		this.map = new google.maps.Map(document.querySelector(this.container), {
+			center: this.position,
+			zoom: this.zoom
+		});
+		
+		this.marker = new google.maps.Marker({
+			position: this.position,
+			map: this.map
+		});	
+	}
+}
+
+const getLocation = async() => {
+  if (navigator.geolocation) {
+    const res = new Promise ((res) => {
+      navigator.geolocation.getCurrentPosition((position) => {
+        const obj =  {
+          lat: position.coords.latitude,
+          lng: position.coords.longitude
+        };
+
+        res(obj);
+      })
+    });
+
+    return await res;
+  } else {
+    alert("Geolocation is not supported by this browser.");
+  }
+}
+
+const handleForm = (event) => {
+  event.preventDefault();
+  
+  if (map && map.position){
+    map.changePosition({lat: +event.target.lat.value, lng: +event.target.lng.value});
+  }
+}
+
+const map = new GoogleMap('.map', getLocation);
+
+export const Location = () => {
+  return (
+    <div className="location">
+      <button onClick={map.initMap}>Show my position</button>
```

Comment:
everytime on click I see an error in console 
```
You have included the Google Maps JavaScript API multiple times on this page. This may cause unexpected errors.
```
http://i.imgur.com/ZJtQClm.png

Reply:
oh really, it's because of I add this map every click
I'll fix it